### DISCUSSION TITLE
[Production] Internal Server Error ...

### DISCUSSION THREAD

**Discussion by martinezguillaume** [vercel/next.js#15128] [Help] [answered]
I've got Internal Server Error only on production...
4 hours that I'm on this problem without success
<img width="518" alt="image" src="https://user-images.githubusercontent.com/17292331/87318723-a371f000-c520-11ea-8cbd-855c21374aef.png">

I don't want you to fix my problem, but I just need advice on how to fix it. I literally have no logs, no errors, no code, just this error displayed.

Naturally, everything is working as expected on local, but as soon as I'm trying to deploy it in Vercel, I've got this. Please help me, I'm so tired

**Comment by jamesmosier** [comment, marked as answer]
Have you checked the ["Function Logs"](https://vercel.com/knowledge/where-are-vercel-logs#function-logs) in Vercel?

![Screen Shot 2020-07-13 at 12 18 56 PM](https://user-images.githubusercontent.com/2854919/87328174-18363180-c503-11ea-80fe-ce1980089170.png)

**Comment by martinezguillaume** [comment]
Oh my.... Thank you man, I see errors in my serverless functions. To be honest, I didn’t even know that I have them. I’ll document my self about that, thank you !!!! ❤️ ❤️

**Reply by pytah100** [reply]
were you able to solve the internal server error.?

**Comment by willin** [comment]
<img width="1038" alt="image" src="https://user-images.githubusercontent.com/1890238/153886515-fb891163-98ce-4ad8-8619-6bcbdc4795f3.png">

Next.js app does not have a log
